masterdata/configuration/drugs/drug.csv

```diff
@@ -515,4 +515,36 @@ Uuid,Void/Retire,Name,Concept Drug,Concept Dosage Form,Strength,_version:1
 ,,Vitamin A 50000 IU,Vitamin A,Oral capsule,50000 IU,
 ,,Vitamin A 100000 IU,Vitamin A,Oral capsule,100000 IU,
 ,,Vitamin A 100000 IU/ml,Vitamin A,Oral solution,100000 IU/ml,
-,,Anti-rabies vaccine,Anti-rabies vaccine,,,
\ No newline at end of file
+,,Anti-rabies vaccine,Anti-rabies vaccine,,,
```

Comment:
Should we remove all other drugs from this CSV file and only retain the drug list provided by GOK (Government of Karnataka)?